testsuite/tests/kuadrant/authorino/wristband/test_wristband.py

```diff
@@ -0,0 +1,27 @@
+"""Edge Authentication tests"""
+from jose import jwt
+
+
+def test_wristband_token_claims(oidc_provider, auth, wristband_token, wristband_endpoint, certificates):
+    """Verify acquired jwt token claims"""
+    wristband_decoded = jwt.decode(wristband_token, certificates["signing_ca"].certificate)
+    assert wristband_decoded["exp"] - wristband_decoded["iat"] == 300
+    assert wristband_decoded["iss"] == wristband_endpoint
+    # check differences in claims between rhsso token and acquired wristband token
+    access_token_decoded = jwt.decode(auth.token.access_token, oidc_provider.get_public_key(), audience="account")
```

Comment:
By using `get_public_key()` without specifically requesting RHSSO, this test will fail if we switch to auth0, as the method is not there